remote_syslog.go

```diff
@@ -53,6 +54,11 @@ func tailOne(file string, excludePatterns []*regexp.Regexp, logger *syslog.Logge
 // Tails files speficied in the globs and re-evaluates the globs
 // at the specified interval
 func tailFiles(globs []string, excludedFiles []*regexp.Regexp, excludePatterns []*regexp.Regexp, interval RefreshInterval, logger *syslog.Logger, severity syslog.Priority, facility syslog.Priority, poll bool) {
+	if len(globs) == 0 {
```

Comment:
I've just answered my own question: yes and no :-) If we check for files earlier on via `Files()`, that should cover files from both the config & CLI, and will allow us to display an error even when damonized. That should make this redundant and allow it to be removed.